Sources/UnionNotifications/IOS18NotificationDialogContent.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  IOS18NotificationDialogContent.swift
+//  union-notifications
+//
+//  Created by RJ Kigner on 9/29/25.
+//
+
+import Foundation
+import SwiftUI
+
+struct  IOS18NotificationDialogContent: View {
+    let appName: String
+    let shouldEnableScroll: Bool
+    let maxContentHeight: CGFloat?
+    
+    private var localizedTitle: String {
+        String(format: String(localized: "\"%@\" Would Like to Send You Notifications", bundle: .module), appName) 
+    }
+    
+    var body: some View {
+        Group {
```

Comment:
`body` uses `@ViewBuilder` by default, so you don't need to wrap this control flow in a `Group`.